influxdb_migration_1_to_timestream_for_influxdb_guide.md

```diff
@@ -0,0 +1,281 @@
+      | access-log-path                    |                                            |
+      | access-log-status-filters          |                                            |
+      | write-tracing                      |                                            |
+      | pprof-enabled                      | pprof-disabled                             |
+      | pprof-auth-enabled                 |                                            |
+      | debug-pprof-enabled                |                                            |
+      | ping-auth-enabled                  |                                            |
+      | https-enabled                      |                                            |
+      | https-certificate                  | tls-cert                                   |
+      | https-private-key                  | tls-key                                    |
+      | shared-secret                      |                                            |
+      | max-row-limit                      |                                            |
+      | max-connection-limit               |                                            |
+      | unix-socket-enabled                |                                            |
+      | bind-socket                        |                                            |
+      | max-body-size                      |                                            |
+      | max-concurrent-write-limit         |                                            |
+      | max-enqueued-write-limit           |                                            |
+      | enqueued-write-timeout             | http-write-timeout                         |
+      |                                    |                                            |
+      | [**logging**]                      |                                            |
+      | format                             |                                            |
+      | level                              | log-level                                  |
+      | suppress-logo                      |                                            |
+      |                                    |                                            |
+      | [**tls**]                          |                                            |
+      | ciphers                            | tls-strict-ciphers                         |
+      | min-version                        | tls-min-version                            |
+      | max-version                        |                                            |
+
+      </details>
+  
+  2. Apply your 1.x custom settings to the comparable InfluxDB 2.7 settings using `influxd` flags, environment variables, or a 2.7 configuration file. For more information about configuring InfluxDB 2.7, see [Configuration options](https://docs.influxdata.com/influxdb/v2/reference/config-options/).
```

Comment:
```suggestion
  2. Apply your 1.x custom settings to the comparable Amazon Timestream for InfluxDB settings using `influxd` flags, environment variables, or a Amazon Timestream for InfluxDB configuration file. For more information about configuring InfluxDB 2.7, see [Configuration options](https://docs.influxdata.com/influxdb/v2/reference/config-options/).
```

Global nit: Shall we replace 2.7 with Amazon Timestream for InfluxDB across the guide (for consistency)?

Reply:
Yes, 2.7 should be replaced with Amazon Timestream for InfluxDB or just Timestream for InfluxDB across the guide.